### PR TITLE
IW-4230 | Apply badges in DiscussionModeration::getReportedPosts

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/IW-4230

@Wikia/iwing

### CODE REVIEW

File: extensions/wikia/DiscussionModeration/ReportedPostsHelper.class.php [hunk 1]
```diff
@@ -90,4 +90,108 @@ private function buildPermalink( Uri $uri ) {
...
+			if ( isset( $post['lastDeletedBy']['id'] ) ) {
+				$userIds[] = (int)$post['lastDeletedBy']['id'];
+			}
+			if ( isset( $post['threadCreatedBy']['id'] ) ) {
+				$userIds[] = (int)$post['threadCreatedBy']['id'];
+			}
+
+			if ( isset( $post['_embedded']['thread'][0]['firstPost']['createdBy']['id'] ) ) {
+				$userIds[] = (int)$post['_embedded']['thread'][0]['firstPost']['createdBy']['id'];
+			}
```
Comment (pstacherzak) [inline extensions/wikia/DiscussionModeration/ReportedPostsHelper.class.php:115]: Does first post have `lastEditedBy` and `lastDeletedBy` fields as well?

Comment (rybmat) [reply extensions/wikia/DiscussionModeration/ReportedPostsHelper.class.php:115]: nope

Comment (rybmat) [reply extensions/wikia/DiscussionModeration/ReportedPostsHelper.class.php:115]: at least I did not see it

Comment (rybmat) [reply extensions/wikia/DiscussionModeration/ReportedPostsHelper.class.php:115]: but according to discussion's code it can, good catch!

**Review by pstacherzak — APPROVED** [review]

**Pull request merged**